app/routers/global_variable.py

```diff
@@ -0,0 +1,33 @@
+from fastapi import APIRouter, Depends, Request
```

Comment:
This shouldn't be here, and should be under demos(create it) because it is not supposed to be on the main app